__tests__/components/home/NewsItem.test.js

```diff
@@ -16,4 +17,14 @@ describe('<NewsItem />', () => {
       expect(node.prop('rel')).toEqual('noopener noreferrer')
     })
   })
+
+  it('renders the currently released version', () => {
+    /*
+      not a concern here, since the input is ours:
+      "Detects RegExp(variable), which might allow an attacker to DOS your server with a long-running regular expression"
+      - https://github.com/nodesecurity/eslint-plugin-security#detect-non-literal-regexp
+     */
+    // eslint-disable-next-line security/detect-non-literal-regexp
```

Comment:
yeah, this is kinda gross, and i am totally open to cleaner suggestions.  i know @ndushay in particular dislikes inline style check exceptions, and this is a particularly bad specimen.

but... i didn't want to disable the rule in general (i think it's a good one), i thought it was totally fine here (per explanation about us controlling the var), and i wanted to explain why, so that people wouldn't copypasta this exception into places where the var contained user input.

🤷‍♂